features/dataset-metadata-citation-doi.feature

```diff
@@ -4,10 +4,10 @@ Feature: Add the metadata schema on dataset page to allow Hypothesis to parse ci
   I want the Hypothesis annotation tool to recognise a GigaDB dataset page url as an alias of its DOI
   So that I can correctly cite it irrespective of which website it appears on or I annotate from
 
-  @ok @javascript @issue-85
+  @wip @javascript @issue-85
   Scenario: Go to dataset page and there is a meta-tag with DOI
     Given I am not logged in to Gigadb web site
-    When I go to "/dataset/100006"
-    Then I should see "Genomic data from Adelie penguin (<em>Pygoscelis adeliae</em>)"
-    And There should be a meta-tag with name "citation_doi" and content "10.5072/100006"
+    When I go to "/dataset/100094"
+    Then I should see "Data and software to accompany the paper: Applying compressed sensing to genome-wide association studies."
+    And There should be a meta-tag with name "citation_doi" and content "10.5072/100094"
```

Comment:
Hi @rija 
I found this `DOI` in the `dataset table` has title with no html tag, so better use this one for this scenario?

Reply:
Hi @kencho51,

It's better to run tests on a known state of database because the state of your local db may change over time as you've noticed. You may have nocticed that some of the acceptance tests have the following step has background, just for that purpose:
```
Background:
	Given Gigadb web site is loaded with "gigadb_testdata.pgdmp" data
```

That way, the tests can expect a stable and predictable state for the database